lib/nerves_hub_link/application.ex

```diff
@@ -22,6 +22,12 @@ defmodule NervesHubLink.Application do
     }
 
     children = [
+      {PropertyTable,
+       name: NervesHubLink,
+       properties: [
+         {["status"], :disconnected},
+         {["update_available"], nil}
```

Comment:
```suggestion
         {["update_available"], :unknown}
```

Reply:
To restate what we discussed on a call, it may be nice for `update_available` to be `true`, `false`, or `:unknown` to avoid a nil value. The nil value could make it seem like something is wrong with the code or the property hasn't been initialized. That's partially true, but this is an expected state while the request is in flight to determine if an update is available or not. Therefore it may be more clear to label this state with an explicit value, like `:unknown`.